topics/bloomfilters.md

```diff
@@ -58,6 +58,88 @@ In this username example, we can use a Bloom filter to track every username that
 * If no, the user is created and the username is added to the Bloom filter.
 * If yes, the app can decide to either check the main database or reject the username.
 
+## Tutorial
+
+### Setup
+
+1. **Install Valkey**: Follow the [installation guides](installation.md) to set up Valkey on your system.
```

Comment:
Why is this needed? it is not needed if someone is using Docker, right?

If you agree, lets remove this